mpqp/core/circuit.py

```diff
@@ -26,13 +26,6 @@
 from typing import TYPE_CHECKING, Iterable, Optional, Sequence, Type
 from warnings import warn
```

Comment:
To continue Hamza's discussion about the problem with cirq, here are minimal examples to show the problem:
```py
>>> c = QCircuit([BasisMeasure(basis=Basis([np.array([0,1]), np.array([1,0])]), shots=0)], nb_qubits=1)
>>> run(c, GOOGLEDevice.CIRQ_LOCAL_SIMULATOR).state_vector
StateVector([-1, 0])
>>> run(c, IBMDevice.AER_SIMULATOR).state_vector
StateVector([0, 1])
```
while 
```py
>>> c = QCircuit([BasisMeasure(basis=Basis([np.array([1,0]), np.array([0,1])]), shots=0)], nb_qubits=1)
>>> run(c, GOOGLEDevice.CIRQ_LOCAL_SIMULATOR).state_vector
StateVector([1, 0])
>>> run(c, IBMDevice.AER_SIMULATOR).state_vector
StateVector([1, 0])
```

Reply:
```py
>>> res = run(c, GOOGLEDevice.CIRQ_LOCAL_SIMULATOR)
>>> res.job.circuit.gphase
3.141592653589793
```

Reply:
It looks like the pre_measure circuit adds a phase, which makes sense, but then I don't get why the result is on the 1st basis vector and not the second one, could it be related to qiskit's qubits ordering deal ?

Reply:
Based on cirq return, I think it has a different internal convention for ordering the states or a global phase that could affect the outcome??? normalization?, phase conventions?, .... investigating!!
While IBM uses the standard computational basis ordering `|0>`, `|1>`

Reply:
should be fixed thanks to julien !

Reply:
We wait for the merge of the solution from `mpqp-to-qasm`